compiler/rustc_ast_pretty/src/pprust/state.rs

```diff
@@ -99,13 +99,13 @@ pub fn print_crate<'a>(
     filename: FileName,
     input: String,
     ann: &'a dyn PpAnn,
-    is_expanded: bool,
+    is_expanded: IsExpanded,
```

Comment:
Rather than having to fill in `.0` suffixes everywhere, what about writing:
```suggestion
    IsExpanded(is_expanded): IsExpanded,
```

Reply:
I prefer it as implemented. These arguments are more frequently passed on to other functions than directly used. You just don't see those lines touched in this PR because the caller and callee have both changed type.

But while a lang team member is reviewing &mdash; `operator bool` when? Or maybe have `if` conditions (and `while` etc) apply `Deref<Target = bool>`, without needing a new trait?

Reply:
Fair enough. I definitely judged based on the lines changed in the diff, and if lots of unchanged lines pass these along, I agree that this wouldn't be an improvement.

(And to answer your other question: gut reaction is that I would rather not have such implicit conversions, but I would also like to understand better when you would expect them to get used apart from self-documenting bool wrappers.)